src/index.html

```diff
@@ -4,14 +4,146 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, user-scalabel=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action ="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset ">
+        <legend>Personal information</legend>
+        <label  class="label">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+          >
+        </label>
+        <label class="label">
+           Name:
+          <input
```

Comment:
```suggestion
        <label class="label">
          Name:
          <input
```
it should be on the same nesting level

Reply:
fix all cases